### PR TITLE
Twitter icon change to X

### PR DESCRIPTION
**Pull request by Harshu467**
image
This is output, 
Merge my PR and issue is #47

### CODE REVIEW

**Comment by Harshu467** [comment]
@ashutosh-rath02 I guess vercel need authorization for deployment?

File: .gitignore [hunk 1]
```diff
@@ -27,7 +27,7 @@ yarn-error.log*
 
 # local env files
 .env*.local
-
+.env
```
Comment (ashutosh-rath02) [inline .gitignore:30]: revert this

File: components/footer.tsx [hunk 2]
```diff
@@ -2,13 +2,16 @@ import React, { ReactNode } from "react";
...
   const currentYear = new Date().getFullYear();
-
+  const TwitterX = () => (
+    <svg xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" width="25" height="25" viewBox="0,0,256,256">
+      <g transform=""><g fill="none" fill-rule="nonzero" stroke="none" stroke-width="none" stroke-linecap="butt" stroke-linejoin="none" stroke-miterlimit="10" stroke-dasharray="" stroke-dashoffset="0" font-family="none" font-weight="none" font-size="none" text-anchor="none" ><path transform="scale(8.53333,8.53333)" d="M24,4c1.105,0 2,0.895 2,2v18c0,1.105 -0.895,2 -2,2h-18c-1.105,0 -2,-0.895 -2,-2v-18c0,-1.105 0.895,-2 2,-2z" id="strokeMainSVG" fill="#190d0d" stroke="#190d0d" stroke-width="3" stroke-linejoin="round"></path><g transform="scale(8.53333,8.53333)" fill="#ffffff" stroke="none" stroke-width="1" stroke-linejoin="miter"><path d="M6,4c-1.105,0 -2,0.895 -2,2v18c0,1.105 0.895,2 2,2h18c1.105,0 2,-0.895 2,-2v-18c0,-1.105 -0.895,-2 -2,-2zM8.64844,9h4.61133l2.69141,3.84766l3.33008,-3.84766h1.45117l-4.12891,4.78125l5.05078,7.21875h-4.61133l-2.98633,-4.26953l-3.6875,4.26953h-1.47461l4.50586,-5.20508zM10.87891,10.18359l6.75391,9.62695h1.78906l-6.75586,-9.62695z"></path></g></g></g>
```
Comment (ashutosh-rath02) [inline components/footer.tsx:12]: Dont place the SVG here. Make a new file in `public/icons/TwitterIcon.svg` , place it there and import

**Review by ashutosh-rath02 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Harshu467) [reply .gitignore:30]: Well It need bcz .env file create should not be push

File: components/footer.tsx (hunk 2 above)
Comment (Harshu467) [reply components/footer.tsx:12]: Well it does not support as ReactNode in other file as it  is void, it will fail

Comment (Harshu467) [reply components/footer.tsx:12]: I did what u said but it gives error for reactNode not support for void

**Comment by Harshu467** [comment]
@ashutosh-rath02 Merge this PR

File: .gitignore (hunk 1 above)
Comment (ashutosh-rath02) [reply .gitignore:30]: Line 29 takes care of that

File: components/footer.tsx (hunk 2 above)
Comment (ashutosh-rath02) [reply components/footer.tsx:12]: Can you search for icons like we did with radix icons? This doesnt look good in code

Comment (Harshu467) [reply components/footer.tsx:12]: Radix does not have new twitter icon , I search their firstly

File: .gitignore (hunk 1 above)
Comment (Harshu467) [reply .gitignore:30]: Well it did  not work on my device !!!!!!!!!

File: components/footer.tsx (hunk 2 above)
Comment (ashutosh-rath02) [reply components/footer.tsx:12]: https://fontawesome.com/icons/x-twitter?f=brands&s=solid

Comment (Harshu467) [reply components/footer.tsx:12]: image

Looks like it is not working

Comment (ashutosh-rath02) [reply components/footer.tsx:12]: You need to debug it dude or find alternatives.😬

**Comment by ashutosh-rath02** [comment]
Resolved

**Pull request closed without merging**